### DISCUSSION TITLE
Works with Google Doorbell?

### DISCUSSION THREAD

**Discussion by VICTORHUGOBRZ** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#811] [Q&A]
Hello. I found out about this solution through a Brazilian technology site. Does anyone know if it is possible to hear my Google Doorbell through this unofficial app?

**Comment by im-coder-lg** [comment]
Well, I don't think so, since it will have to use the Internet continuously. It could be possible, but I think the SDK limits the functionality.
